argobots.spec

```
@@ -24,12 +24,6 @@ BuildRequires: autoconf, automake, libtool
 # need libunwind if configuring symbolic ULTs stack dumps feature
 BuildRequires: libunwind-devel
 
```

Adding valgrind to the configure options needs the `valgrind-devel` package:

```suggestion
BuildRequires: valgrind-devel

```

---

Shouldn't it be "valgrind-dev" in argobots/debiam/control then ?

---

No.  https://packages.ubuntu.com/focal/amd64/valgrind/filelist.  The parity between -devel and -dev are not always 1:1.